### DISCUSSION TITLE
Guardians audit medium finding question ( spoiler! )

### DISCUSSION THREAD

**Discussion by bbl4de** [Cyfrin/security-and-auditing-full-course-s23#53] [Q&A]
**DO NOT READ THIS QUESTION IF YOU DIDN'T COMPLETE SECTION 8!**

Hi, 
I have just finished auditing the guardians audit codebase and compared my findings with the "answer key" in audit-data. 
I found myself confused with the M-1 finding - "potentially incorrect voting period and delay in governor may affect governance". Where can I find the information that the default return for the `votingPeriod` and `votingDelay` functions is the block number and not the timestamp? I looked at OpenZeppelin's docs for the Governor contract and even EIP6372 proposal docs without success. How could I have found this bug by myself in this codebase? I missed something or is it some more in-depth knowledge about the libraries?

**Comment by PatrickAlphaC** [comment]
Let's goo!!!!! We have more to update in section 8 actually! 

Have you tried looking for those keywords in solodit? https://solodit.xyz/
